packages/camera/camera_avfoundation/ios/Classes/CameraPlugin.m

```diff
@@ -169,25 +174,10 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call result:(FlutterResult)re
           [weakSelf.registry textureFrameAvailable:cameraId];
         }
       };
-      FlutterMethodChannel *methodChannel = [FlutterMethodChannel
-          methodChannelWithName:
-              [NSString stringWithFormat:@"plugins.flutter.io/camera_avfoundation/camera%lu",
-                                         (unsigned long)cameraId]
-                binaryMessenger:_messenger];
-      FLTThreadSafeMethodChannel *threadSafeMethodChannel =
-          [[FLTThreadSafeMethodChannel alloc] initWithMethodChannel:methodChannel];
-      _camera.methodChannel = threadSafeMethodChannel;
-      [threadSafeMethodChannel
-          invokeMethod:@"initialized"
-             arguments:@{
-               @"previewWidth" : @(_camera.previewSize.width),
-               @"previewHeight" : @(_camera.previewSize.height),
-               @"exposureMode" : FLTGetStringForFLTExposureMode([_camera exposureMode]),
-               @"focusMode" : FLTGetStringForFLTFocusMode([_camera focusMode]),
-               @"exposurePointSupported" :
-                   @([_camera.captureDevice isExposurePointOfInterestSupported]),
-               @"focusPointSupported" : @([_camera.captureDevice isFocusPointOfInterestSupported]),
-             }];
+      _camera.dartAPI = [[FCPCameraEventApi alloc]
+          initWithBinaryMessenger:_messenger
+             messageChannelSuffix:[NSString stringWithFormat:@"%ld", cameraId]];
+      [_camera reportInitializationState];
```

Comment:
I moved the actual channel invocation into `FLTCam` because it seemed strange to have it here when a) the camera instance already has the channel, and b) all of the values here come from the camera instance.